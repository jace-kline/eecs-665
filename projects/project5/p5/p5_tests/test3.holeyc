bool f(int a) {
    return (a > 0);
}

int main() {
    bool p;
    int x;
    TOCONSOLE true + "hello";
    x = 1 + f(true);
    x = 1 + f(1,2);
    return 3 + true;
}