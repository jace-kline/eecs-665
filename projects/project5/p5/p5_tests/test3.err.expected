FATAL [8,15]: Arithmetic operator applied to invalid operand
FATAL [8,22]: Arithmetic operator applied to invalid operand
FATAL [9,15]: Type of actual does not match type of formal
FATAL [9,13]: Arithmetic operator applied to invalid operand
FATAL [10,13]: Function call with wrong number of args
FATAL [10,13]: Arithmetic operator applied to invalid operand
FATAL [11,16]: Arithmetic operator applied to invalid operand
Type Analysis Failed
